rest-api/model/participant_summary.py

```diff
@@ -26,6 +25,11 @@
 # queries that don't ask for withdrawn participants.
 WITHDRAWN_PARTICIPANT_VISIBILITY_TIME = datetime.timedelta(days=2)
 
+# suspended participants don't allow contact but can still use samples. These fields
+# will not be returned when queried on suspended participant.
+SUSPENDED_PARTICIPANT_FIELDS = ['zipCode', 'city', 'streetAddress', 'phoneNumber',
```

Comment:
Where did this list of fields come from? I didn't see it in the attached ticket or document.

Reply:
+1. Barring any requirements otherwise, I'd assume we can return everything for suspended participants... it's only withdrawn participants that we need to filter stuff out for.

Reply:
There was no strict requirement to remove those fields specifically. But there is a no_contact requirement similar to withdrawn. I just found it prudent, if you both disagree I'll remove it.

Reply:
My understanding is withdrawn means "I'm not in the program anymore -- stop using my data". So after 48 hours, their data  (apart from a few fields) disappears from everywhere.

Suspended just means "don't contact me". So there's no reason we should filter their data out. Though you should confirm with HPRO folks.

Basically I think this requirement is a no-op; we already support what they want.

Reply:
OK. Spoke with the PM team for RDR and HPRO, they do want this left in.